ios/GoodDollar/Info.plist

```diff
@@ -2,6 +2,19 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>branch_key</key>
+	<string>key_live_ibOUGFCo6HgZzzHZpAJQNnbgBxoABsxI</string>
```

Comment:
the same question as https://github.com/GoodDollar/GoodDAPP/pull/1413/files#r386992922

Reply:
The file is a config file but the keys should be a secret. How do we handle this?

Reply:
see here check if possible to pass keys in subscribe method. I think pr says initializing needs to be set to false in .json
in any case secrets should not be committed.https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/pull/405

Reply:
I researched and its not possible through subscribe.
only through branch.json